topo/bond/forch/faucet.yaml

```diff
@@ -0,0 +1,121 @@
+dps:
```

Comment:
why is there a separate faucet.yaml file here? There's already one in faucet/faucet.yaml -- something here likely needs to be adjusted (e.g. dynamically copy the file at runtime) rather than copying the entire file.

Reply:
So there has to be a structural config file and a behavioral config file. By 'copy the file at runtime', do you mean it has to be copied in the setup_stack script?

Reply:
Ok, I think we should remove the faucet/faucet.yaml and keep forch/faucet.yaml and Forch will copy it to faucet/ directory. I kept both because in the past Faucet has to be started before Forch, which requires a faucet.yaml to exisit at the begining. Since we are running everything in a single controller container, we do not need such a file anymore and Forch will create it...